src/selectors/getAllProjects.js

```diff
@@ -1,8 +1,12 @@
+import isNil from 'lodash-es/isNil';
 import sortBy from 'lodash-es/sortBy';
 import values from 'lodash-es/values';
 import {createSelector} from 'reselect';
 
 export default createSelector(
   state => state.get('projects'),
-  projects => sortBy(values(projects.toJS()), project => -project.updatedAt),
+  projects =>
+    sortBy(values(projects.toJS()), ({updatedAt}) =>
+      isNil(updatedAt) ? -Number.MAX_VALUE : -updatedAt,
```

Comment:
This seemed like quickest fix, but I'm not sure if using Number.MAX_VALUE is appropriate in this context.

Reply:
Oh cool! That is very useful!